src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java

```diff
@@ -1001,6 +1000,7 @@ public String parseHarvestingClient(JsonObject obj, HarvestingClient harvestingC
         harvestingClient.setMetadataPrefix(obj.getString("metadataFormat",null));
         harvestingClient.setHarvestingSet(obj.getString("set",null));
         harvestingClient.setCustomHttpHeaders(obj.getString("customHeaders", null));
+        harvestingClient.setAllowHarvestingMissingCVV(obj.getBoolean("allowHarvestingMissingCVV", false));
```

Comment:
@stevenwinship One quick thing I can think of - we could use an equivalent line in JsonPrinter.java, to show this setting in the output of the GET version of the API, when present.

Reply:
added